doc/lightning-waitinvoice.7.txt

```diff
@@ -12,14 +12,21 @@ SYNOPSIS
 
 DESCRIPTION
 -----------
-The *waitinvoice* RPC command waits until a specific invoice is paid, then
-returns that single entry as per *listinvoice*.
+The *waitinvoice* RPC command waits until a specific invoice is paid,
+then returns that single entry as per *listinvoice*.
 
 RETURN VALUE
 ------------
-On success, an invoice description will be returned as per lightning-listinvoice(7): 'complete' will always be 'true'.
+On success, an invoice description will be returned as per lightning-listinvoice(7).
+The 'status' field will indicate whether the invoice was paid or was
+expired.
```

Comment:
This change is now wrong; on success 'complete' will still always be 'true'.

Reply:
Oops